telegram/commands/report/tooth.go

```diff
@@ -0,0 +1,66 @@
+package report
+
+import (
+	"activity-tracker/shared"
+	"activity-tracker/storage"
+	"fmt"
+	"time"
+)
+
+var (
+	reportToothMessage = `
+	%s - Esta semana te has cepillado por día esta cantidad 🪥🪥:
+
+	Lunes: %d
+	Martes: %d 
+	Miércoles: %d 
+	Jueves: %d
+	Viernes: %d
+	Sábado: %d
+	Domingo: %d
+
+	Con estos números se avecina una caries ni la hp, ponete juicios@ pues 🤢
+	`
+)
+
+// SendToothReport sends the tooth report
+func SendToothReport(bot *shared.Bot, userName, content string, chatID int64) error {
+	wr, err := GenerateToothReport(bot, userName, chatID)
```

Comment:
whats wr?

Reply:
same of above jsjs wr --> water report :v it must be modified xd

Reply:
fixed